### PR TITLE
tickets/DM-32942

### PR DESCRIPTION
**Pull request by mareuter**

### CODE REVIEW

File: charts/uws-api-server/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,10 @@ apiVersion: v1
...
-version: "1.3.1"
+version: "1.4.0"
 home: https://github.com/lsst-dm/uws-api-server
 maintainers:
-  - name: Timothy Andrew Manning
-    email: manninga@illinois.edu
+  - name: Kian-Tat Lim
+    email: ktl@slac.standford.edu
+  - name: Michael Reuter
+    email: mareuter@lst.org
```
Comment (ktlim) [inline charts/uws-api-server/Chart.yaml]:
```suggestion
    email: ktl@slac.stanford.edu
  - name: Michael Reuter
    email: mareuter@lsst.org
```

File: charts/uws-api-server/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Universal Worker Service API Server
+
+**Homepage:** <https://github.com/lsst-dm/uws-api-server>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Kian-Tat Lim | ktl@slac.standford.edu |  |
+| Michael Reuter | mareuter@lst.org |  |
```
Comment (ktlim) [inline charts/uws-api-server/README.md]:
```suggestion
| Kian-Tat Lim | ktl@slac.stanford.edu |  |
| Michael Reuter | mareuter@lsst.org |  |
```

File: charts/uws-api-server/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# uws-api-server
```
Comment (ktlim) [inline charts/uws-api-server/README.md:1]: If this is auto-generated, should it be checked in?

**Review by ktlim — APPROVED** [review]

File: charts/uws-api-server/README.md (hunk 3 above)
Comment (mareuter) [reply charts/uws-api-server/README.md:1]: It is manually generated from the `helm-docs` program. Haven't heard if SQuaRE found an automatic process yet.

File: charts/uws-api-server/Chart.yaml (hunk 1 above)
Comment (mareuter) [reply charts/uws-api-server/Chart.yaml]: Fixed.

File: charts/uws-api-server/README.md (hunk 2 above)
Comment (mareuter) [reply charts/uws-api-server/README.md]: Fixed.

**Pull request merged**